.github/workflows/dockers-agent-ngt-image.yaml

```
@@ -274,4 +274,15 @@ jobs:
     uses: ./.github/workflows/_docker-image.yaml
     with:
       target: agent-ngt
+<<<<<<< HEAD
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 277: could not find expected ':'

(syntax-check)